tests/test_search.py

```diff
@@ -154,14 +190,92 @@ def test_elasticsearch_exception(self):
         es_logger.setLevel("ERROR")
         try:
             os.environ['DSS_ES_ENDPOINT'] = "bogus"
-            response = self.post_search(smartseq2_paired_ends_query,
-                                        requests.codes.internal_server_error)
-            self.assertEqual(response.json['code'], "elasticsearch_error")
-            self.assertEqual(response.json['title'], "Elasticsearch operation failed")
+            url = self.build_url()
+            self.assertPostResponse(
+                path=url,
+                json_request_body=dict(es_query=smartseq2_paired_ends_query),
+                expected_code=requests.codes.internal_server_error,
+                expected_error=ExpectedErrorFields(code="internal_server_error",
+                                                   status=requests.codes.internal_server_error))
```

Comment:
Not a blocking issue, but in general, internal server error stands for an unexpected error. In this case, we know what caused the error: it was a malformed query. So it should really be a 400 series error. We should file this as a separate issue to fix.

Reply:
created issues #547 to correct error.